godtools/App/Features/SetupParallel/SetupParallelLanguageView.swift

```diff
@@ -45,63 +45,66 @@ class SetupParallelLanguageView: UIViewController {
         
         super.viewDidLoad()
         
+        tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(handleSelectLanguageTapped))
+        tapRecognizer.cancelsTouchesInView = false
+        tapRecognizer.delegate = self
+        self.view.addGestureRecognizer(tapRecognizer)
+        
         setupLayout()
         setupBinding()
     }
     
+    override func viewWillDisappear(_ animated: Bool) {
+        
+        self.view.removeGestureRecognizer(self.tapRecognizer)
+        self.tapRecognizer.delegate = nil
+    }
+    
+    private func setupLayout() {
+        
+        promptLabel.text = viewModel.promptText
+                
+        yesButton.setTitle(viewModel.yesButtonText, for: .normal)
+        
+        noButton.setTitle(viewModel.noButtonText, for: .normal)
+        
+        getStartedButton.setTitle(viewModel.getStartedButtonText, for: .normal)
+        
+        setupSelectLanguageButton()
+        setupBottomButtons()
+    }
+    
     private func setupBinding() {
         
         animatedView.configure(viewModel: viewModel.animatedViewModel)
         
-        selectLanguageButtonGestureRecognizer.addTarget(self, action: #selector(handleSelectLanguageTapped))
-        
         yesButton.addTarget(self, action: #selector(handleYesTapped), for: .touchUpInside)
```

Comment:
Move addTargets into the viewDidLoad() just because we keep the setupBinding() relative to the viewModel.